.git-blame-ignore-revs

```diff
@@ -0,0 +1,4 @@
+# .git-blame-ignore-revs
+
+# Applied automated eslint autofixes and prettier to map.es6
+be3a9b6041e451b02c671220be50b5ca33cae1a8
```

Comment:
This entry prevents [be3a9b6] from dominating the [online git-blame view for map.es6](https://github.com/kobotoolbox/kpi/blame/968da59c86307ad2afe929c41bf7b8c101418caa/jsapp/js/components/map.es6).

Reference:
- https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
- https://www.michaelheap.com/git-ignore-rev/

Internal discussion:

- https://chat.kobotoolbox.org/#narrow/stream/4-Kobo-Dev/topic/Prettier.2FESLint.20Autofixes/near/238951